packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -55,8 +55,8 @@ const tessenAction =
       );
     }
 
-    const customerId = JSON.parse(Cookies.get('ajs_user_id') || 'null');
-    const anonymousId = JSON.parse(Cookies.get('ajs_anonymous_id') || 'null');
+    const customerId = Cookies.get('ajs_user_id') || null;
+    const anonymousId = Cookies.get('ajs_anonymous_id') || null;
```

Comment:
don't think this is a biggie, but this string has encoded quotes in it `%22` which makes querying in NR a little awkward. i can't recall if it always has that escaped quote chars or not.

```
...and anonymousId = '"5188d5e8-e919-4dbe-84d4-17046a7236dc"' since...
```